doc/020_installation.rst

```diff
@@ -100,6 +102,29 @@ On OpenBSD 6.3 and greater, you can install restic using ``pkg_add``:
 
     # pkg_add restic
 
+.. _official_binaries:
+
+Official Binaries
+*****************
+
+Stable Releases
+===============
+
+You can download the latest stable release versions of restic from the `restic
+release page <https://github.com/restic/restic/releases/latest>`__. These builds
+are considered stable and releases are made regularly in a controlled manner.
+
+There's both pre-compiled binaries for different platforms as well as the source
+code available for download. Just download and run the one matching your system.
+
+Unstable Builds
+===============
+
+Another option is to use the latest builds for the master branch, available on
+the `restic beta download site <https://beta.restic.net>`__. These too are
```

Comment:
You could use the following link, then the latest build is always shown on to:

https://beta.restic.net/?sort=time&order=desc